### PR TITLE
A very good branch to fix bugs and refactor

### PR DESCRIPTION
**Pull request by ZaitzeV16**
some minor fixes

### CODE REVIEW

File: src/test/java/com/greenfoxacademy/chamaaragonite/kingdom/KingdomControllerTest.java [hunk 1]
```diff
@@ -42,13 +42,11 @@
...
       MediaType.APPLICATION_JSON.getSubtype(),
       StandardCharsets.UTF_8);
-  // endregion Fields
-
 
   // region MockBeans
   @MockBean
   private KingdomService kingdomService;
-  // endregion MockBeans
+  // endregion MockBeans// endregion Fields
```
Comment (mrkvcstms) [inline src/test/java/com/greenfoxacademy/chamaaragonite/kingdom/KingdomControllerTest.java]:
```suggestion
  // endregion MockBeans
  // endregion Fields
```

**Comment by ZaitzeV16** [comment]
@csadesz 
FYI +206 −233 
😄

File: src/main/java/com/greenfoxacademy/chamaaragonite/building/buildingtypes/Academy.java [hunk 2]
```diff
@@ -12,6 +12,7 @@
 
   // region Constructors
   public Academy() {
+    this.setLevel(1);
```
Comment (csadesz) [inline src/main/java/com/greenfoxacademy/chamaaragonite/building/buildingtypes/Academy.java]: We can add this level 1 as default at the Building default constructor

**Comment by ZaitzeV16** [comment]
very much refactor, very deletion

**Pull request merged**